src/forum-tool-selector-settings/ForumToolSelectorSettings.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import ForumToolSelectorContainer from './ForumToolSelectorSettings/ForumToolSelectorContainer';
+import StudioAPISevice from '../data/services/StudioApiService';
+
+function tempGetForumToolFeaturesList(data) {
+  let features = [];
+  for (let i = 0; i < data.length; i++) {
+    for (let j = 0; j < data[i].length; j++) {
+      features = features.concat(data[i][j].features);
+    }
+  }
+  const unique = [...new Set(features)];
+  return unique;
+}
+
+function ForumToolSelectorSettings({ courseId }) {
+  const [paginatedForumList, setPaginatedForumList] = useState(null);
+  const [featuresList, setFeaturesList] = useState(null);
+  const [selectedId, setSelectedId] = useState(null);
+
+  useEffect(() => {
+    const data = StudioAPISevice.dummyForumData(courseId);
+    setPaginatedForumList(data);
+    setFeaturesList(tempGetForumToolFeaturesList(data));
+  }, []);
+
+  const onSelected = (id) => {
+    setSelectedId(id);
+  };
+  const onUnselected = () => {
+    setSelectedId(null);
+  };
+
+  if (!paginatedForumList) {
+    return (<p>Loading...</p>);
+  }
+
+  return (
+    <ForumToolSelectorContainer
+      paginatedForumList={paginatedForumList}
+      featuresList={featuresList}
+      selectedForumName={selectedId}
+      onSelected={onSelected}
+      onUnselected={onUnselected}
+    />
+  );
+}
```

Comment:
As I explained in our call remove the useEffect from here and directly pass in the data. In the future, we will add the code to load it from the API.